src/bodywork/cli/workflow_jobs.py

```diff
@@ -45,11 +45,11 @@ def create_workflow_job_in_namespace(
     if not k8s.namespace_exists(namespace):
         print(f"namespace={namespace} could not be found on k8s cluster")
         return None
-    if _is_existing_workflow_job(namespace, job_name):
+    if job_name and _is_existing_workflow_job(namespace, job_name):
```

Comment:
I'm not sure you need the `and` if `job_name` cannot be `None` - i.e. if it isn't `Optional[str]`.

Reply:
I have no idea why I put that there.